R/cluster_rs.R

```diff
@@ -97,7 +104,7 @@ cluster_rs_probabilities <-
       prob_unit <- tapply(prob_unit, INDEX = clusters, FUN = unique)
     }
     if (!is.null(n_unit)) {
-      n <- unique(n_unit)
+      n_unit <- tapply(n_unit, INDEX = clusters, FUN = unique)
```

Comment:
add simplify = false, otherwise you'll have a bug when all clusters have only one thing.